projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-component.scss

```diff
@@ -673,6 +686,11 @@
             @extend %igx-grid__hierarchical-expander--cosy !optional;
         }
 
+        @include e(hierarchical-expander, $m: empty) {
```

Comment:
If this is an additive class, you don't have to extend the placeholder selector for each density.